### DISCUSSION TITLE
地下城盔甲无法用铁砧附魔[redirected]

### DISCUSSION THREAD

**Discussion by mottledart** [Rad586/Forget-Me-Not#74] [Q&A] [answered]
每次将附魔书摆上去就会闪退
整合包版本2.7.0
游戏版本1.19.2
启动器pcl
[错误报告-2024-6-25_20.06.42.zip](https://github.com/user-attachments/files/15970959/-2024-6-25_20.06.42.zip)

**Comment by Rad586** [comment, marked as answer]
确认问题了，见https://github.com/Rad586/Forget-Me-Not/issues/75
Confirmed and redirected to https://github.com/Rad586/Forget-Me-Not/issues/75

**Reply by Rad586** [reply]
2.7.1已修复
Fixed in 2.7.1
